### PR TITLE
feat: added initial testing platform

### PR DESCRIPTION
**Pull request by galuszkak**

### CODE REVIEW

**Comment by galuszkak** [comment]
The code is passing CI:
https://github.com/galuszkak/backend/actions/runs/188256882

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -67,7 +82,7 @@ jobs:
 
       - uses: actions/setup-node@v1
         with:
-          node-version: 13
+          node-version: 10
```
Comment (TomaszMolenda) [inline .github/workflows/ci.yml:85]: Why did you change this line? Each step is independent and can have different environment.

Comment (galuszkak) [reply .github/workflows/ci.yml:85]: Hi @TomaszMolenda,
I wanted to unify node-version that is aligned with node version that is used by Google Cloud Functions. GCP Cloud Functions are using Node 10.

To remove uncertainty if the feature is available in Node 10 or not (and by accident not use a feature from Node 13), it makes sense, to just use same version of NodeJS. 

Same things I did with `@types/node`.

If that doesn't make sense, let me know.

Comment (thecodersio) [reply .github/workflows/ci.yml:85]: I think it's a good idea to keep the same version.

Comment (thecodersio) [reply .github/workflows/ci.yml:85]: > Hi @TomaszMolenda,
> I wanted to unify node-version that is aligned with node version that is used by Google Cloud Functions. GCP Cloud Functions are using Node 10.
> 
> To remove uncertainty if the feature is available in Node 10 or not (and by accident not use a feature from Node 13), it makes sense, to just use same version of NodeJS.
> 
> Same things I did with `@types/node`.
> 
> If that doesn't make sense, let me know.

You're right. I think it's a good idea to keep compatibility version with GCP.

**Comment by thecodersio** [comment]
Hey @galuszkak 

I've changed the base to the `release/4.1.2` branch 

Please take a look on it and resolve the conflicts :pray: 

Thanks!

**Comment by galuszkak** [comment]
@thecodersio done.

**Comment by galuszkak** [comment]
@thecodersio test is failing because someone removed that module. I will remove then test too.

**Comment by galuszkak** [comment]
@thecodersio It looks in this case my PR doesn't add any value add all. 

I think it can be then closed. It's over a month hanging and it doesn't make sense now to add this if there is already in this release testing platform. It would make more sense if the development would be more open source and this would be incremental small change. Right now it doesn't add anything substantial.

**Comment by thecodersio** [comment]
@galuszkak 

I agree with you 100% - we can close this PR.

I'm sorry that our development process looks like this.
I see a lot of problems related to this here and will bring it up in the next retrospective.
